src/static-data/rules/rule_set_templates/__nds_kita_szenarien.yml

```diff
@@ -0,0 +1,26 @@
+# laut beschluss vom 4. mrz (kita in nds)
+
+__nds_kita_szenarien:
+      
+  - conditions:
+      date_from: '2021-03-08'
+      incidence_to: 100
+      incidence_days: 3
+    references:
+      - date: '4.3.2021'
+        link: 'https://www.mk.niedersachsen.de/startseite/aktuelles/presseinformationen/kitas-offnen-am-8-marz-in-festen-gruppen-schrittweise-weitere-schuloffnungen-ab-15-marz-tonne-grosser-schritt-nach-vorne-bei-den-kleinsten-bis-ostern-besuchen-alle-schulerinnen-und-schuler-ihre-schule-198040.html'
+    rules:
```

Comment:
das wird eine sehr lange zeile in der app.
ich würde das trennen:
[open] **Alle** Kinder dürfen in die Betreuung
[limit] Ggf. Einschränkungen beim bringen & holen
[limit] Keine Gruppenmischung